test/standard/formatter_test.rb

```diff
@@ -26,7 +26,7 @@ def test_no_uncorrected_offenses_prints_nothing
 
   def test_does_not_print_fix_command_if_run_with_fix
     @subject = Standard::Formatter.new(@io, auto_correct: true, safe_auto_correct: true)
-    @subject.file_finished(@some_path, [Offense.new(false, 42, 13, "Neat")])
+    @subject.file_finished(@some_path, [Offense.new(false, 42, 13, "Neat", "Bundler/InsecureProtocolSource")])
```

Comment:
Since we're leveraging all the `Cop` logic from Rubocop itself, I had to supply real `cop_name`s.